src/style-sheets/pages/_contact-us-view.scss

```diff
@@ -55,6 +55,14 @@
                 width: 131px;
                 align-self: flex-end;
             }
+            & input {
+                font-family: inherit;
+                font-style: inherit;
+            }
+            & textarea {
```

Comment:
Also can you add more padding to the top of the textarea to resemble the input. You can do this on the Textarea component itself. should find that sass in _textarea.scss

Reply:
Looks good!